src/player.js

```diff
@@ -72,20 +91,23 @@ export default class Player {
           } = tralbumDetails.tracks[i];
           const type = "t";
 
+          if (!is_purchasable) return;
```

Comment:
previously this was lower and the responsibility of skipping an item was shared by this check AND in `createOneClickBuyButton` -- but it turns out it is sort of stupid to pass a boolean argument to a function and instead making it this layers responsibility makes it way more clear what is going on.